server/src/GraphQL/Queries/Inspector.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Gernzy\Server\GraphQL\Queries;
+
+use \App;
+use Gernzy\Server\Exceptions\GernzyException;
+use GraphQL\Type\Definition\ResolveInfo;
+use Illuminate\Support\Facades\File;
+use Nuwave\Lighthouse\Support\Contracts\GraphQLContext;
+
+class Inspector
+{
+    /**
+     * Return a value for the field.
+     *
+     * @param  null  $rootValue Usually contains the result returned from the parent field. In this case, it is always `null`.
+     * @param  mixed[]  $args The arguments that were passed into the field.
+     * @param  \Nuwave\Lighthouse\Support\Contracts\GraphQLContext  $context Arbitrary data that is shared between all fields of a single query.
+     * @param  \GraphQL\Type\Definition\ResolveInfo  $resolveInfo Information about the query itself, such as the execution state, the field name, path to the field from the root, and more.
+     * @return mixed
+     */
+
+    public function index($rootValue, array $args, GraphQLContext $context, ResolveInfo $resolveInfo)
+    {
+        return true;
+    }
+
+
+    public function packages($rootValue, array $args, GraphQLContext $context, ResolveInfo $resolveInfo)
+    {
+        $file = __DIR__ . "/../../../composer.json";
```

Comment:
@AlexeiaW would we be able to use `base_path` here?
https://laravel.com/docs/7.x/helpers#method-base-path

Reply:
Thank you, base_path() works well! I always struggle with file paths...